examples/gallery/histograms/scatter_and_histograms.py

```diff
@@ -0,0 +1,68 @@
+"""
+Scatter plot with histograms
+----------------------------
+To create a scatter plot with histograms at the sides of the plot one
+can use :meth:`pygmt.Figure.plot` in combination with
+:meth:`pygmt.Figure.histogram`. The positions of the histograms are plotted
+by offseting them from the main scatter plot figure using
+:meth:`pygmt.Figure.shift_origin`.
```

Comment:
Not completely happy with this formulation. Do you feel this suggestion is an improvement?
```suggestion
To create a scatter plot with histograms at the sides of the plot one
can use :meth:`pygmt.Figure.plot` in combination with
:meth:`pygmt.Figure.histogram`. The positions of the histograms are set
relative to the scatter plot in the center using
:meth:`pygmt.Figure.shift_origin`.
```

Reply:
OK. Then we leave it as is, since I have no better idea for now 🙂.